### PR TITLE
Fixes #12677 - Nic::Interface attr_accessible :ip should be on Base

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: app/models/nic/base.rb [hunk 1]
```diff
@@ -7,13 +7,10 @@ class Base < ActiveRecord::Base
...
-                    :mac, :name, :type,
-                    :provider, :username, :password,
-                    :identifier, :virtual, :link, :tag, :attached_to,
-                    :managed, :bond_options, :attached_devices, :mode,
-                    :primary, :provision, :compute_attributes,
-                    :_destroy # used for nested_attributes
+    attr_accessible :attached_to, :attached_devices, :bond_options,
+      :compute_attributes, :host_id, :host, :identifier, :ip, :link, :mac,
+      :managed, :mode, :name, :password, :primary, :provider, :provision, :type,
+      :tag, :username, :virtual, :_destroy # used for nested_attributes
```
Comment (ares) [inline app/models/nic/base.rb]: the comment applies for `:_destroy` only which is not clear now, anyway I think you can remove it, I think everyone knows that...

**Comment by ares** [comment]
except the comment nit :+1:

**Comment by dLobatog** [comment]
Moved the _destroy comment to a new line to clarify it only applies to it.

**Comment by ares** [comment]
s/Fix/Fixes/ in commit message please

**Comment by dLobatog** [comment]
Updated, sorry!

**Comment by ares** [comment]
Merged as 97fda1ccd608532bcf703e1ffa9ac6f876433424, thanks @dLobatog!

**Pull request closed without merging**